minecraft_servers/mineleague/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+   "server_name":"mineleague",
+   "nice_name":"MineLeague.eu",
+   "direct_ip":"mineleague.eu",
+   "supported_languages":[
+	  "de",
+	  "en",
+	  "al"
+   ],
+   "social":{
+      "web":"https://mineleague.eu",
+      "discord":"https://link.mineleague.eu/discord"
```

Comment:
As custom discord invite links are only supported for LabyMod partners at the moment.
Please use a discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url